packages/itwinui-css/src/file-upload/file-upload.scss

```diff
@@ -74,3 +75,56 @@
     }
   }
 }
+
+@mixin iui-file-card {
+  width: 100%;
+  height: 100%;
+  line-height: normal;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  > svg,
+  .iui-file-card-icon {
+    @include iui-icon-style(xl);
+    @include iui-icon-color;
+    margin-top: var(--iui-size-s);
+    margin-bottom: var(--iui-size-s);
+    margin-left: var(--iui-size-xs);
+  }
+
+  .iui-browse-input {
+    width: 0.1px;
```

Comment:
`iui-browse-input` looks odd. this is a new component, so a new class should be used

it would be better to use visually-hidden styles instead of this code.

Reply:
This has been changed to a new component with the visually-hidden styling